.github/workflows/release.yml

```diff
@@ -30,8 +30,8 @@ jobs:
       - name: Publish app
         working-directory: ./src/ProfanityFilter.Action
         run: |
-          dotnet publish /t:PublishContainer -c Release \
-            -o ./publish \
+          dotnet publish
```

Comment:
I think there's a problem here (because I did the same thing) - need a trailing `\` on this line like the others have.